docs/blog/things-to-know-when-developing-with-lit-element/index.md

```diff
@@ -0,0 +1,233 @@
+
+---
+
+## Default Values
+
+If you'd like to define default values for a property on a LitElement based custom element, there are a number of ways that you could do this. While some do require management across the entire element class, we'll ignore those today as there are a number of options that do not require such work.
+
+First off, there's a very direct comparison between LitElement and a functional approach to UI development where you could consider the `render()` method of a LitElement as an almost 1 to 1 conversion from the functional definitions found in other offerings. In this way, you could assume `render()` is the only entry into your properties and leverage your fallbacks like so:
+
+```js
+render() {
+  const closeDelay = this.closeDelay ?? 300;
+  ...
+}
+```
+
+From here, you could get very close to simply copy and paste the rest of a functional UI component into a LitElement for an early test of its capabilities. Leveraging scoped variables like this, rather than class properties, does leave you in much the same place that functional alternatives to UI development do; needing additional tools for memoization of that scope. As using LitElement means you're already in a class context, we can use the capabilities of a class directly rather than synthesizing them with memoization.
+
+A simpler approach to merging the two concepts is to set that fallback into the class property itself. Above I showed doing this in the `render()` lifecycle method, however, I find it much nicer to _only_ have the template surfaced therein. Leveraging one of the earlier lifecycle methods for managing defaults, validation, sanitation, transformation, or derivation help to maintain that structure. In this case, we'll use `willUpdate()` which doesn't require a `super` call or a returned value, but will always be visited during each render lifecycle:
+
+```js
+willUpdate(changedProperties: PropertyValues<this>): void {
+  this.closeDelay = this.closeDelay ?? 300;
+}
+```
+
+With this approach, you can most closely facilitate the single line defaulting that can be present when relying on a function to define a component. This does mean the `shouldRender()` the first method in the render lifecycle will not have your default value, and if that's an issue for your style of element development, you might want to move this fallback work there, but it also means that due to the fact the LitElement renders asynchronously there's technically a possibility that your local methods could be as well. Due to this fact, you may want to leverage a slightly more complete approach to a default.
+
+Here we see the property getter fallback to the value:
+
+```js
+get closeDelay(): number {
+  return this._closeDelay || 300;
+}
+set closeDelay(value: number) {
+  const closeDelay = value;
+  if (closeDelay === this.closeDelay) return;
+  this.requestUpdate('closeDelay', this.closeDelay);
+  this._closeDelay = closeDelay;
+}
+_closeDelay = 300;
```

Comment:
Would it be better to mark this private (I never used it, but I see if can be done in JS with `#closeDelay`), or is implicit private ('_') good enough?

Reply:
could add a section here on how to write a 'defaulting property' decorator with typescript

Reply:
Honestly not sure than devs in most other contexts know what `#closeDelay` would mean in this case, and think it distracts from the broader point to try to describe it. 

There are lots of other sections that could be included here, and I think the right call is to rename the article "Some things..." to imply there is always more to know!